### PR TITLE
Pdu Aten

### PR DESCRIPTION
**Pull request by marcin-usielski**

### CODE REVIEW

File: moler/cmd/pdu_aten/pdu/read_status.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Command read status..
```
Comment (Ernold11) [inline moler/cmd/pdu_aten/pdu/read_status.py:3]: double dot at the end

File: moler/device/pdu_aten.py [hunk 2]
```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+"""
+PduAten device class
```
Comment (Ernold11) [inline moler/device/pdu_aten.py:3]: no dot at the end

File: moler/cmd/pdu_aten/pdu/generic_pdu.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+from moler.cmd.pdu_aten.generic_pdu_aten import GenericPduAten
+from moler.exceptions import CommandFailure
+
+__author__ = 'Marcin Usielski'
+__copyright__ = 'Copyright (C) 2020, Nokia'
+__email__ = 'marcin.usielski@nokia.com'
+
+cmd_failure_causes = ['not found',
+                      'Invalid command',
+                      ]
```
Comment (Ernold11) [inline moler/cmd/pdu_aten/pdu/generic_pdu.py:18]: move close bracket to line 17

File: moler/cmd/pdu_aten/pdu/read_status.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+__email__ = 'marcin.usielski@nokia.com'
+
+
+class ReadStatus(GenericPdu):
+
+    def __init__(self, connection, outlet, output_format=None, prompt=">", newline_chars=None, runner=None):
+        """
+        Class for command read status for PDU Aten device.
+
+        :param connection: connection to device.
```
Comment (tkrol88) [inline moler/cmd/pdu_aten/pdu/read_status.py:21]: lack outlet and output_format in doc strings

File: moler/cmd/pdu_aten/pdu/read_status.py [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+>"""
+
+COMMAND_KWARGS = {
+    "outlet": "o01",
+    "output_format": "format",
+}
+
+COMMAND_RESULT = {
+    "STATUS": 'on',
+    "OUTLET": {
```
Comment (tkrol88) [inline moler/cmd/pdu_aten/pdu/read_status.py:81]: do we need OUTLET key? I think that STATUS is enough

File: moler/cmd/pdu_aten/pdu/read_status.py (hunk 1 above)
Comment (marcin-usielski) [reply moler/cmd/pdu_aten/pdu/read_status.py:3]: done

File: moler/device/pdu_aten.py (hunk 2 above)
Comment (marcin-usielski) [reply moler/device/pdu_aten.py:3]: Done

File: moler/cmd/pdu_aten/pdu/generic_pdu.py (hunk 3 above)
Comment (marcin-usielski) [reply moler/cmd/pdu_aten/pdu/generic_pdu.py:18]: Done

File: moler/cmd/pdu_aten/pdu/read_status.py (hunk 5 above)
Comment (marcin-usielski) [reply moler/cmd/pdu_aten/pdu/read_status.py:81]: Maybe future software will support printing status for many outlets?

Comment (tkrol88) [reply moler/cmd/pdu_aten/pdu/read_status.py:83]: hmm, so then STATUS key will not be necessary :P At this moment is support only one outlet so I think we should keep only STATUS key. If something change in the future then new key will be added

Comment (marcin-usielski) [reply moler/cmd/pdu_aten/pdu/read_status.py:83]: For simple output command returns only STATUS but for format with value 'format' it returns STATUS and dict with outlet id. So if anyone doesn't want dict in return value then please use simple format.

Comment (tkrol88) [reply moler/cmd/pdu_aten/pdu/read_status.py:83]: OK :)

**Pull request merged**